src/internal/components/dnd-area/styles.scss

```diff
@@ -0,0 +1,70 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+@use '../../styles' as styles;
+@use '../../styles/tokens' as awsui;
+@use '@cloudscape-design/component-toolkit/internal/focus-visible' as focus-visible;
+
+@mixin animated {
+  @include styles.with-motion {
+    transition: transform awsui.$motion-duration-transition-quick awsui.$motion-easing-transition-quick;
+  }
+}
+
+@mixin drag-overlay-with-radius($border-radius) {
+  border-start-start-radius: $border-radius;
```

Comment:
Shouldn't border radius styles be defined in user land rather than by the DndArea component?

Reply:
The DndArea uses className that the consumer assigns to sortable items to set the following styles:
1. Box shadow of the active item;
2. Focus outline of the active item;
3. Dragging item placeholder overlay;
4. Sortable items animation.

The box shadow, focus outline, and placeholder overlay styles require border-radius and those cannot inherit the border-radius of the item because are applied as separate absolute-positioned elements.